modules/drivers/sqlite/src/metabase/driver/sqlite.clj

```diff
@@ -77,6 +79,19 @@
          ;; disallow "FDW" (connecting to other SQLite databases on the local filesystem) -- see https://github.com/metabase/metaboat/issues/152
          {:limit_attached 0}))
 
+(defmethod driver/describe-table-indexes :sqlite
+  [driver database table]
+  (let [pk (first (sql-jdbc.execute/do-with-connection-with-options
+                   driver database nil
+                   (fn [conn]
+                     (sql-jdbc.describe-table/get-table-pks :sqlite conn (:name database) table))))]
+    ;; In sqlite a PK will implicitly have a UNIQUE INDEX, but if the PK is integer the getIndexInfo method from
+    ;; jdbc doesn't return it as indexed. so we need to manually get mark the pk as indexed here
+    (cond-> ((get-method driver/describe-table-indexes :sql-jdbc) driver database table)
+      (string? pk)
```

Comment:
Why `string?`? What other type could `pk` be?
```suggestion
      (string? pk)
```

Reply:
nil

Reply:
Then ideally you should use `some?` instead. When I read this, I was questioning why such a specific check was made, and wondering whether the `pk` could be something other than a string. It doesn't help that `get-table-pks`'s docstring doesn't mention the return type.